src/main/kotlin/dev/arbjerg/ukulele/audio/Player.kt

```diff
@@ -58,6 +58,8 @@ class Player(val beans: Beans, guildProperties: GuildProperties) : AudioEventAda
     val isPaused : Boolean
         get() = player.isPaused
 
+    var isLooping : Boolean = false
```

Comment:
Might wanna add a getter like `isPaused`

Reply:
The getter is implicit. `isPaused` only has an explicit getter because it is not a real variable